src/commands/cacert/export.ts

```diff
@@ -41,23 +39,23 @@ export default class Export extends Command {
 
   async run() {
     const { flags } = this.parse(Export)
-    const ctx: any = {}
+    const kube = new KubeHelper(flags)
     const cheHelper = new CheHelper(flags)
-    const cheTasks = new CheTasks(flags)
-    const apiTasks = new ApiTasks()
-    const tasks = new Listr([], { renderer: 'silent' })
 
-    tasks.add(apiTasks.testApiTasks(flags, this))
-    tasks.add(cheTasks.verifyCheNamespaceExistsTask(flags, this))
+    if (!await kube.hasReadPermissionsForNamespace(flags.chenamespace)) {
+      throw new Error(`E_PERM_DENIED - Permission for Che server namespace "${flags.chenamespace}" are required`)
+    }
+    if (!await cheHelper.cheNamespaceExist(flags.chenamespace)) {
+      throw new Error(`E_BAD_NS - Namespace ${flags.chenamespace} does not exist. Please specify it with --chenamespace flag`)
```

Comment:
Could you fix the error here
https://github.com/che-incubator/chectl/blob/master/src/tasks/che.ts#L496
`The configuration cannot ..` should be removed.